app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,5 @@
 <resources>
   <string name="app_name">Danger Android Sample</string>
+
+  <string name="unused_text">text</string>
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/droibit/danger-android-sample/blob/9695fd0e52135600b4d6d3f13bcd1a956238c991/app/src/main/res/values/strings.xml#L4"></span>The resource <code>R.string.unused_text</code> appears to be unused</td>
    </tr>
  </tbody>
</table>

Reply:
I fixed.